### PR TITLE
readme: update an outdated doc link

### PR DESCRIPTION
**Pull request by dcalvin**
Signed-off-by: Calvin Weng <wenghao@pingcap.com>

###  What have you changed?

Please explain in detail what the changes are in this PR and why they are needed:

- updated a stale link that referenced the wiki 

Don't assume reviewers understand the original issue.

###  What is the type of the changes?

Pick one of the following and delete the others:
 
- Misc (other changes)

###  How is the PR tested?
Please select the tests that you ran to verify your changes:
 - No code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ When a node starts, the metadata of the Node, Store and Region are recorded into
 
 TiKV was originally a component of [TiDB](https://github.com/pingcap/tidb). To run TiKV you must build and run it with PD, which is used to manage a TiKV cluster. You can use TiKV together with TiDB or separately on its own.
 
-We provide multiple deployment methods, but it is recommended to use our Ansible deployment for production environment. The TiKV documentation is available on [TiKV's wiki page](https://github.com/tikv/tikv/wiki/).
+We provide multiple deployment methods, but it is recommended to use our Ansible deployment for production environment. The TiKV documentation is available on [TiKV's website](https://tikv.org/docs/3.0/).
```
Comment (siddontang) [inline README.md]: seem `https://tikv.org/docs/3.0` can't be accessed.

Comment (dcalvin) [reply README.md]: Weird. I remember I tested it. Anyways, now fixed.

**Pull request merged**